src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -107,6 +107,7 @@ public void CopyTo_MultidimensionalArray_ThrowsIndexOutOfRangeException()
         }
 
         [Fact]
+        [ActiveIssue(17462, TargetFrameworkMonikers.UapAot)]
```

Comment:
Ok. I was just curious because I saw the rest of them being Uap only. Thanks for disabling them!